### ISSUE TITLE
Request for a new release

### ISSUE THREAD

**Issue by meghprkh** [Scony/godot-gdscript-toolkit#314]
Hi, thanks for the toolkit!

I would like setting up a pre-commit hook for my project, but that breaks due to the `pkg_resources` / `setuptools` requirement in Python3.12. Would it be possible to release a new version with e3c37fa in it?

Note currently this does work

```
pre-commit autoupdate --bleeding-edge --repo https://github.com/Scony/godot-gdscript-toolkit
```

**Comment by Scony** [comment]
Yes, it will be released soon - along with Godot 4.3 release.

**Comment by Scony** [comment]
Done
